README.md

```diff
@@ -226,5 +227,12 @@ activeActive: # edit values according to your cluster
         fqdn: <cluster3_name>.<cluster3_namespace>.svc.cluster.local
 ```
 
+#### IPV4 enforcement
+You might not have IPV6 support in your K8S cluster.
+In this case, you could enforce the use of IPV4, by adding attribute to the REC spec:
```

Comment:
It's just a typo, it should be
"... by adding the following attribute to the REC spec"

Reply:
ok